### PR TITLE
Disable unreliable test

### PR DESCRIPTION
**Pull request by StephenLPeters**
This test is failing at a rate which is blocking other check ins, disabling until we can put eyes on it.

### CODE REVIEW

**Comment by StephenLPeters** [comment]
I've opened issue #1815 to track

File: dev/Repeater/APITests/RepeaterTests.cs [hunk 1]
```diff
@@ -193,7 +193,7 @@ public void ValidateGetSetBackground()
             });
         }
 
-        [TestMethod]
+        //[TestMethod] disabled with issue #1815
```
Comment (ranjeshj) [inline dev/Repeater/APITests/RepeaterTests.cs]: This was unreliable only in specific configurations, so perhaps we could still have it running in others ?

Comment (kaiguo) [reply dev/Repeater/APITests/RepeaterTests.cs]: Yeah, we can probably just revert this https://github.com/microsoft/microsoft-ui-xaml/pull/1797, which will skip debug builds.

**Pull request merged**